db/compaction/compaction_iterator_test.cc

```diff
@@ -1079,7 +1080,7 @@ INSTANTIATE_TEST_CASE_P(CompactionIteratorWithAllowIngestBehindTestInstance,
 class CompactionIteratorTsGcTest : public CompactionIteratorTest {
  public:
   CompactionIteratorTsGcTest()
-      : CompactionIteratorTest(test::ComparatorWithU64Ts()) {}
+      : CompactionIteratorTest(rocksdb::ComparatorWithU64Ts()) {}
```

Comment:
Other things aside, we no longer hard-code the namespace name as `rocksdb` any more.

Reply:
removed it.